401-code-challenges/array-shift/array-shift.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+function insertShiftArray(array, insert){
+  if (!Array.isArray(array)){
+    return 'I need an array to put something into'}
+  else if (typeof insert !== 'number' && typeof insert !== 'string'){
```

Comment:
I don't believe the code challenge specified that the insert value needed to be a number or string. I personally would allow any value, and think it is confusing to have `insertShiftArray` only take strings and numbers. Nothing about that name makes me think it would only allow a certain data type and would probably cause confusion.